devnet/gssmr-ecs/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+services: 
+  alice:
+    image: 500822580415.dkr.ecr.us-east-2.amazonaws.com/gssmr-devnet-alice
```

Comment:
Shouldn't they all use the same image with different runtime settings (aka env variables)?

Reply:
Can't we have the `gossamer --init` step done at runtime then i.e. `gossamer --init ... && ...`? :thinking: 

Maybe a bit out of scope in this PR though.

Reply:
For the Alice one I have to do it first, and swap out the `node.key` file.  I guess it doesn't have to be the same for `bob.Dockerfile` but not sure we want to to add that step slowing down the launch.